### PR TITLE
PD-442 missing license events and PD-446 describe the `scalesize` parameter

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/autopilot/reference/_index.md [hunk 1]
```diff
@@ -102,6 +102,11 @@ This action is to perform expansion on Portworx Storage Pools.
 * **scaletype**: Specifies the type of operation to be performed to expand the pool. Supported values are:
     * *add-disk*: Portworx will add new disk(s) to the existing storage pool
     * *resize-disk*: Portworx will resize existing disks in the storage pool
+* **scalesize**:  Specifies the amount, in GiB or TiB, by which Autopilot should exapnd a storage pool.
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/autopilot/reference/_index.md]: type in `exapnd`

File: content/portworx-install-with-kubernetes/autopilot/reference/_index.md [hunk 2]
```diff
@@ -115,6 +120,16 @@ Expand the pool by 50% of current size by adding disks
       scaletype: "add-disk"
 ```
 
+Expand the pool by 100GiB by resizing disks
+
+```text
+  actions:
+  - name: openstorage.io.action.storagepool/expand
+    params:
+      scalesize: "100GiB"
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/autopilot/reference/_index.md]: Sorry, I think this needs to be `100Gi` (and not `100GiB`).

Comment (nathan-px) [inline content/portworx-install-with-kubernetes/autopilot/reference/_index.md]:
```suggestion
      scalesize: "100Gi"
```

File: content/portworx-install-with-kubernetes/autopilot/reference/_index.md [hunk 3]
```diff
@@ -102,6 +102,11 @@ This action is to perform expansion on Portworx Storage Pools.
 * **scaletype**: Specifies the type of operation to be performed to expand the pool. Supported values are:
     * *add-disk*: Portworx will add new disk(s) to the existing storage pool
     * *resize-disk*: Portworx will resize existing disks in the storage pool
+* **scalesize**: Specifies the amount, in GiB or TiB, by which Autopilot should expand a storage pool.
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/autopilot/reference/_index.md]: this needs to be `Gi or Ti`

```suggestion
* **scalesize**: Specifies the amount, in Gi or Ti, by which Autopilot should expand a storage pool.
```

File: content/portworx-install-with-kubernetes/autopilot/reference/_index.md [hunk 4]
```diff
@@ -115,6 +120,16 @@ Expand the pool by 50% of current size by adding disks
       scaletype: "add-disk"
 ```
 
+Expand the pool by 100GiB by resizing disks
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/autopilot/reference/_index.md]:
```suggestion
Expand the pool by 100Gi by resizing disks
```

**Pull request merged**